pup-0001.md

```diff
@@ -85,31 +87,29 @@ additional commits to the pull request either as patches or additional pull requ
 
 ### Voting
 
+The lazy consensus model is used for deciding via votes on the mailing list.
+
 The +1, +0, -0, -1 voting process is used; votes are sent as responses via the
-announcement/discussion mailing list. The voting has the following meanings:
+announcement/discussion thread on the mailing list. The voting has the following meanings:
 
 * +1: "Will benefit the project and should definitely be adopted."
 * +0: "Might benefit the project and is acceptable."
 * -0: "Might not be the right choice but is acceptable."
-* -1: "Not the right choice and should definitely not be adopted." This type of feedback must come
-  with a statement of objective concerns explaining what harm will come from adopting the proposal.
-
-Note that votes can come from anyone, not just core devs [1]. This is directly modeled after [the
-Django process](https://docs.djangoproject.com/en/dev/internals/contributing/bugs-and-features/#how-we-make-decisions)
-who modeled their process after the Python and Apache communities.
+* -1: "Not the right choice and should definitely not be adopted."
 
 
-### Deciding
+### Accepting/Rejecting
 
-* If an obvious consensus arises via voting, the PUP is ratified.
-* Discussion on the email thread can confirm if consensus is obvious or not.
-* Votes come from anyone, not just core devs [1].
-* A -1 vote from a core dev [1] is considered a veto vote and will block ratification until the
-  vote is recast.
-* Core devs are expected to consider and echo serious feedback from the community, especially any -1
-  votes from community members.
-* Consensus is not always possible. If obvious consensus is not reached, then the core devs [1]
-  decide.
+* A PUP is accepted with two, +1 votes from core devs [1] and no blocking (-1) votes from core devs.
+* +/- 0 votes inform support or dissent but do not pass or block a pup. The stated concerns may 
+  influence other people to have blocking reservations.
+* A -1 vote from a core dev [1] is considered a blocking vote and must be accompanied with
+  substantive arguments.
+* Stated concerns with a blocking vote can be used to revise a PUP until the blocking vote is
+  recast.
+* Votes can come from anyone, not just core devs [1] and are used as data points.
+* Core devs [1] are expected to consider and echo serious feedback from the community, especially
```

Comment:
I like the idea that we seriously consider feedback from the community but what does echo mean here?